src/providers/test/get-transaction-count.test.ts

```diff
@@ -1,47 +1,55 @@
-import { ethers } from 'ethers';
-import Web3 from 'web3';
-import { jsonRpcProvider } from '../..';
+import * as unfetch from 'isomorphic-unfetch';
+import { jsonRpcProvider, tinyBig } from '../..';
 import type { BlockTag } from '../../types/Block.types';
+import { mockOf } from './mock-of';
 import { rpcUrls } from './rpc-urls';
+import { buildFetchInit, buildRPCPostBody } from '../../classes/utils/fetchers';
+
+jest.mock('isomorphic-unfetch');
 
-// coinbase 1 hotwallet
 const address = '0x71660c4005ba85c37ccec55d0c4493e66fe775d3';
-/**
- *
- * @param rpcUrl
- * @param blockTag
- */
```

Comment:
I removed this comment you had above the helper to stay consistent with the other test helpers that don't have a comment like this, but lmk if you want me to go through an add a comment like this above all the helpers used in tests.